rules/game_process.tex

```diff
@@ -340,6 +364,19 @@ \subsection{Free Kick}
 Note that in the case of the Pushing free kick the sub-state is activated automatically through the ``Foul''.
 The team who is awarded the free kick (termed the attacking team) has \qty{\FreeKickTime}{\second} to complete the kick.
 
+In Champions Cup, the referee communicates which team is to take the free kick via a visual gesture.
+The GameController packet will tell the robots which free kick sub-state is active, but not who is to kick the ball.
+\unsure{This gesture is entirely tentative and just my (Francesco's) proposal, pending discussion with the TC. The intent is to mirror the FIFA gestures in a simplified way.}
+The gesture consists of raising one arm horizontally and sideways, parallel to the field line, while the other arm remains at rest.
+The raised arm is the one pointing towards the defending team's goal, i.e. the goal of the team that is \textit{not} taking the free kick.
+\textit{(For reference, the gesture is the same as the "Kick-in" gesture from the 2023 Visual Referee Challenge, but the opposite arm is raised with respect to back then.)}
+This gesture is made by the head referee and held for at least 5 seconds after the GameController signal is sent.
+\textit{(The referee may look at the GameStateVisualizer monitor to check that the GC signal is sent and when the time allotted for the gesture has passed.)}
+The referee sends the signal standing as close as possible to the T-junction, but their actual positioning along the sideline may deviate from this standard position, even significantly,
+e.g. if the referee needs to better observe a specific situation during the game.
+
+In Challenge Shield, this visual signal is not performed and the kicking team is communicated by the GameController.
```

Comment:
Last year with the indirect kick there was a subsection for champions cup and one for challenge shield and this could be handled this year in the same way?

Reply:
Well, the only difference is this one paragraph about the gesture, unlike the indirect kick where the whole rule changes, so having a CC subsection and a CS subsec might be unnecessary.
But I think I can add just one "Visual gesture" subsection that starts with a message like "This only applies to Champions Cup", so the part that differs between the two divisions is still separate from the rest. It would also make the cross-reference from the penalty kick section more precise.